content/overview/getting-started/_index.en.md

```diff
@@ -5,50 +5,149 @@ weight: 2
+```shell
+wash> ctl start provider wasmcloud.azurecr.io/httpserver:0.10.0
+```
+Let's take a look at our host's inventory now, you can either type the command above again or use the `UP/DOWN` arrow keys to navigate to the previous command. It should look something like this (some output truncated for documentation):
+```shell
+               Host Inventory (NBMRMKELSYA645ZB...)                           
+                                                                                                                              
+hostcore.osfamily                                           unix                                                              
+hostcore.os                                                 macos                                                             
+hostcore.arch                                               x86_64                                                            
+repl_mode                                                   true                                                              
+                                                                                                                              
+Actor ID       Image Reference                                                   
+MBCFOPM6JW2... wasmcloud.azurecr.io/echo:0.1.0                                         
+                                
+Provider ID    Link Name        Image Reference                        
+VDHPKGFKDI3... default          N/A                                    
+VAHNM37G4AR... __wasmcloud_...  N/A                                    
+VAG3QITQQ2O... default          wasmcloud.azurecr.io/httpserver:0.10.0 
+```
 
-# Call to action is default enabled, if you want to disable it. just change the 
-enable = false
+#### Linking Actors and Capability Providers
+Now your `Echo` Actor and `HTTP Server` Providers are running, but they aren't connected. In order to allow the Actor and Provider to communicate, we need to `link` them together, which we do using their respective Public Keys (shown in the Inventory as `Actor ID` and `Provider ID`).
 
-####### Default parameters ###########
-[params]
-logo = "images/logo.png"
-# Meta data
-description = "This is meta description"
-author = "Themefisher"
-# contact form action
-contact_form_action = "#" # contact form works with https://formspree.io
-# Google Analitycs
-google_analitycs_id = "" # your id
+You can use the following command to `link` your Actor and Provider, on MacOS you might receive a message asking if you want `wash` to receive incoming network connections. You should click "Allow", this is just setting up the provider's local HTTP server to listen on port 8080.

```

Comment:
your actor and provider. On macOS you might...

Reply:
Might want to be explicit that the dialog box on the mac is creating a temporary firewall exception.